src/managers/input/InputManager.cpp

```diff
@@ -216,6 +216,8 @@ void CInputManager::mouseMoveUnified(uint32_t time, bool refocus) {
         g_pCompositor->setActiveMonitor(PMONITOR);
 
     if (g_pSessionLockManager->isSessionLocked()) {
+        const bool notFirstFocus = g_pCompositor->m_pLastFocus;
```

Comment:
what if going from focused window/layer/whatever -> session lock?

Reply:
For the same reason as above, `g_pCompositor->m_pLastFocus` is null after the session is locked so this will always be false until the lock surface is focused for the first time.